src/client.cpp

```diff
@@ -297,28 +298,14 @@ void ADDON_Announce(const char *_UNUSED(flag), const char *sender,
  * PVR Client AddOn specific public library functions
  ***********************************************************/
 
-const char* GetPVRAPIVersion(void)
-{
-  static const char *apiVersion = XBMC_PVR_API_VERSION;
-  return apiVersion;
-}
-
-const char* GetMininumPVRAPIVersion(void)
-{
-  static const char *minApiVersion = XBMC_PVR_MIN_API_VERSION;
-  return minApiVersion;
-}
-
```

Comment:
Also here, the both functions must be there otherwise the function table bring a crash during call from KODI